Modules/Core/Common/include/itkNeighborhoodOperator.h

```diff
@@ -87,6 +87,12 @@ class ITK_TEMPLATE_EXPORT NeighborhoodOperator : public Neighborhood<TPixel, VDi
   /** Slice iterator type alias support */
   using SliceIteratorType = SliceIterator<TPixel, Self>;
 
+  using PixelRealType = typename NumericTraits<TPixel>::RealType;
+
+  /** Typedef support for coefficient vector type. Necessary
```

Comment:
typedef or alias? I thought we chose to use alias consistently.

Reply:
Good catch. Changed in d7c9182.